site/content/api/Command-And-Control-API.md

```diff
@@ -85,25 +69,4 @@ The status property must contain a valid HTTP status code:
 
```

Comment:
what is the code returned if the device is not connected? what about 503 ?

Reply:
Could that happen? So the device is disconnected after sending the command and before the adapter has send it to the device - in my picture this part is in "sync" so if the adapter could not forward the command the disposition would not be successful.

Reply:
this seems to be part of the question of how an application can know, if its request has reached the device or not. With our current implementation strategy, the application would not get any credits on its sender link, right? If this is the *specified behavior* then we need to put it into the spec.